packages/templates/src/test/raw.js

```diff
@@ -20,25 +20,27 @@
 import { readdirSync, readFileSync } from 'fs';
 import { resolve, basename } from 'path';
 
-/**
- * Internal dependencies
- */
-import { PAGE_LAYOUT_TYPES } from '../../../edit-story/components/library/panes/pageLayouts/constants';
+const PAGE_LAYOUT_TYPES = {
```

Comment:
Really don't want us to maintain this list manually here.

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/6252/commits/be7959947a36ae6a0eb6f2e267bd5c694abbb65e